docs/Release_notes.md

```diff
@@ -1,6 +1,30 @@
 CARMA Platform Release Notes
 ----------------------------
 
+Version 3.3.0, released January 31st, 2020
+--------------------------------------------------------
+
+**Summary:**
+CARMAPlatform release version 3.3.0 is comprised of two key enhancements. First the plugin manager is now able to return a list of services that match a user-requested capability. Second, there are new configurations for the Freightliner Cascadia, including lidar configurations which allow autoware to combine the two lidar devices' data into a single stream. 
```

Comment:
Change the following sentence from 
"configurations which allow autoware to"
to
"configurations which leverage the autoware points_concat node to"

We only use select components from autoware in the system, so its best not to refer to it as a singular entity in this way.